src/panels/Menu/TopMenuBar/Menus/ViewMenu.tsx

```diff
@@ -128,28 +142,58 @@ export function ViewMenu() {
       >
         <Menu.Label>
           <Group justify={'space-between'}>
-            Set visibility level
+            Visibility level
             <InfoBox>
               {`Controls what settings will be exposed in the interface. Increase the
                 level to reveal more advanced settings.`}
             </InfoBox>
           </Group>
         </Menu.Label>
-        <Radio.Group
-          value={propertyVisibility?.toString()}
-          onChange={(newValue) => setPropertyVisibility(parseInt(newValue))}
-          onKeyDown={(event) => event.stopPropagation()}
-        >
-          <Stack gap={'xs'} m={'xs'}>
-            {userLevelOptions ? (
-              Object.entries(userLevelOptions).map(([key, option]) => (
-                <Radio key={key} aria-label={option} label={option} value={key} />
-              ))
-            ) : (
-              <LoadingBlocks />
-            )}
-          </Stack>
-        </Radio.Group>
+        <Container>
+          <Radio.Group
+            value={propertyVisibility?.toString()}
+            onChange={(newValue) => setPropertyVisibility(parseInt(newValue))}
+            onKeyDown={(event) => event.stopPropagation()}
+          >
+            <Stack gap={'xs'}>
+              {userLevelOptions ? (
+                Object.entries(userLevelOptions).map(([key, option]) => (
+                  <Radio key={key} aria-label={option} label={option} value={key} />
+                ))
+              ) : (
+                <LoadingBlocks />
+              )}
+            </Stack>
+          </Radio.Group>
+        </Container>
+
+        <Menu.Label>
+          <Group justify={'space-between'}>
```

Comment:
For consistency with other info boxes, maybe we should place this info box next to the title?

Reply:
I agree, and think we should do this everywhere TBH.  

Here I just copied the code for the already existing visibility label, but I'll remove the space between for that one as well